library/classes/Installer.class.php

```diff
@@ -1157,6 +1165,10 @@ public function quick_install()
             if (! $this->install_gacl()) {
                 return false;
             }
+
+            if (! $this->install_additional_users()) {
```

Comment:
need to also support this in the manual install in setup.php. will basically place the block to support this after line 1231 in setup.php

Reply:
When is setup.php called versus the installer in the docker setup?  I want to make sure I can test this.

Reply:
there is a openemr-cmd to reset the openemr install without doing a autoconfigure.
```
openemr-cmd dr
```
Then can install manually via setup.php
(note there are some particular settings needed when installing via setup.php; Server Host is `mysql`, User Hostname is `%`)
(this is something i can also test)

Reply:
btw, the command to then do an autoinstall (nice to have admin/pass combo for testing) is:
```
openemr-cmd dri
```
and then if want to do an autoinstall with the standard demo data (this is cool because it then tests the sql upgrade script):
```
openemr-cmd drid
```

Reply:
I should of mentioned that there are commands to also create/use a snapshot (if don't want to lose openemr data and go back to something after testing above; it backs up and restores the openemr data from drive and database). if needed, can even collect it off the docker ("capsule") to use on docker rebuilds.

Reply:
I knew about the drid, haven't used the dr or dri commands.  I'll have to look at the snapshot stuff as that will be useful.  Thanks.